manifests/init.pp

```diff
@@ -11,11 +11,16 @@
   $ca_crt,
   $cert,
   $key,
+  $ta            = undef,
   $dh            = undef,
   $mode          = 'client', # client or server
+  $protocol      = 'udp',
+  $cypher        = undef, # Specify specific CYPHER if required.
```

Comment:
Yeh I don't understand why the specific difference was made. If the rendered config file contains a entry for `cipher`, how does it help the end user for the module param to be different?